dashboard/db/migrate/20151013224202_create_hint_view_requests.rb

```diff
@@ -0,0 +1,13 @@
+class CreateHintViewRequests < ActiveRecord::Migration
+  def change
+    create_table :hint_view_requests do |t|
+      t.references :user, index: true, foreign_key: true
+      t.references :script, index: true, foreign_key: true
+      t.references :level, index: true, foreign_key: true
+      t.integer :feedback_type
+      t.string :feedback_xml
```

Comment:
Hmm, that's probably not guaranteed. Do we have a standard for our string sizes?